subsys/bluetooth/host/gatt.c

```diff
@@ -1553,6 +1553,7 @@ static u8_t disconnected_cb(const struct bt_gatt_attr *attr, void *user_data)
 			/* Clear value if not paired */
 			if (!bt_addr_le_is_bonded(conn->id, &conn->le.dst)) {
 				bt_addr_le_copy(&cfg->peer, BT_ADDR_LE_ANY);
+				cfg->id = 0U;
```

Comment:
I suggest we move these to something like:
static void clear_ccc_cfg(struct bt_gatt_ccc_cfg *cfg)
{
    	bt_addr_le_copy(&cfg->peer, BT_ADDR_LE_ANY);
	cfg->id = 0U;
	cfg->value = 0U;
        memset(cfg->data, 0, sizeof(cfg->data));
}

Or just memset the entire thing since that appear to do the same thing.

Reply:
Fixed

Reply:
I used `clear_ccc_cfg` as `BT_ADDR_LE_ANY` is defined outside and may change in the future.